### ISSUE TITLE
Can't get DLSS debug display HUD overlay to show up in Dying Light 2

### ISSUE THREAD

**Issue by TheSpider12** [emoose/DLSSTweaks#89]
As the title says. Anyone managed to make the debug overlay to appear in this game? Tried OverrideDlssHud = 1 and OverrideDlssHud = 2 but still nothing. Dlss files are version 3.1.30, game is version 1.11.3. Tested with another game (Spider-Man Remastered) and the overlay was displayed just fine there.

**Comment by emoose** [comment]
Hmm think that used to work before, IIRC the games blur settings had an effect on the overlay text, maybe some other setting is also affecting it now?
Could be worth checking with the [dev DLL](https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/dev/nvngx_dlss.dll) + [reg file](https://raw.githubusercontent.com/NVIDIA/DLSS/main/utils/ngx_driver_onscreenindicator.reg) to see if that works too.

**Comment by Lewison78** [comment]
DLSS debug display HUD is also not working in Chernobylite, tried every possible combination with Dev DLSS and DisableDevWatermark = true, as well as OverrideDlssHud set to 2, in the latest version of dlsstweaks (200.7 beta 2). Btw, the DisableDevWatermark, does nothing in this game, i dont know why

**Comment by emoose** [comment]
DL2 seems to show the overlay for me fine, both with the default DLSS DLL it came with & the 3.1.30 version:

![23-08-18_21-10-46-374_DyingLightGame_x64_rwdi](https://github.com/emoose/DLSSTweaks/assets/1755499/58243918-1803-4be7-9c6f-7b103e3578f1)

Reg file & OverrideDlssHud both seemed okay for me, that's on steam ver though, maybe you're on a different ver or have some setting making it hide?

@Lewison78 hmm, so chernobylite wouldn't display HUD on any of the 0.200 vers, but 0.123 does?
If you didn't already maybe try changing chernobylite's nvngx_dlss.dll to the 3.1.30 from https://www.techpowerup.com/download/nvidia-dlss-dll/, the 0.200.7beta3 from https://github.com/emoose/DLSSTweaks/issues/88#issuecomment-1684184758 should be able to work with it.

**Comment by Lewison78** [comment]
@emoose First of all, thank you for your response. Second of all, you are correct, HUD is not displayed on any of the 0.200 versions, from what I have tested. Also, from what I have noticed, I was able to switch presets (A, B, C, D, F) on the 0.200 version, on the go, only when using dlss' dev versions, such as this one: https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/dev/nvngx_dlss.dll, every non dev version, that is also included on the techpowerup's website, wont let me swap presets. However this problem does not exist on the 0.123 . I will keep you posted about the 0.200.7beta3 as soon as I can

**Comment by TheSpider12** [comment]
> DL2 seems to show the overlay for me fine, both with the default DLSS DLL it came with & the 3.1.30 version:
> 
> ![23-08-18_21-10-46-374_DyingLightGame_x64_rwdi](https://user-images.githubusercontent.com/1755499/261704524-58243918-1803-4be7-9c6f-7b103e3578f1.png)

Yeah I try DL2 again today and the overlay displays just fine now. Weird. 

Btw, is it possible to swap preset on the go, as in while the game is opened, we can change the preset in the ini file from default to F for example and it is immediately shown in the overlay? For DL2 I have to restart the game for the changed preset to take effects.

**Comment by Lewison78** [comment]
> @emoose First of all, thank you for your response. Second of all, you are correct, HUD is not displayed on any of the 0.200 versions, from what I have tested. Also, from what I have noticed, I was able to switch presets (A, B, C, D, F) on the 0.200 version, on the go, only when using dlss' dev versions, such as this one: https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/dev/nvngx_dlss.dll, every non dev version, that is also included on the techpowerup's website, wont let me swap presets. However this problem does not exist on the 0.123 . I will keep you posted about the 0.200.7beta3 as soon as I can

![image](https://github.com/emoose/DLSSTweaks/assets/106681577/5d2ae45c-5186-4eb7-8625-42e78492277a)
beta3 is working perfectly on the latest dev DLSS

**Comment by emoose** [comment]
> Btw, is it possible to swap preset on the go, as in while the game is opened, we can change the preset in the ini file from default to F for example and it is immediately shown in the overlay? For DL2 I have to restart the game for the changed preset to take effects.

If you setup the dev DLSS dll (https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/dev/nvngx_dlss.dll) then the overlay will let you change preset with "ctrl+alt+]" key, sadly it's not included in the normal DLSS ver though, and dev DLL might have small performance impact IIRC.

Editing dlsstweaks.ini during runtime will try to tell DLSS about the preset changes too, but doesn't always let it change them properly (sometimes you might need to go into game options and turn off DLSS & turn it back on for it to set itself up again)

**Comment by TheSpider12** [comment]
> > Btw, is it possible to swap preset on the go, as in while the game is opened, we can change the preset in the ini file from default to F for example and it is immediately shown in the overlay? For DL2 I have to restart the game for the changed preset to take effects.
> 
> If you setup the dev DLSS dll (https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/dev/nvngx_dlss.dll) then the overlay will let you change preset with "ctrl+alt+]" key, sadly it's not included in the normal DLSS ver though, and dev DLL might have small performance impact IIRC.
> 
> Editing dlsstweaks.ini during gameplay will also try to tell DLSS about the preset changes too, but that doesn't always let it change them properly (sometimes you might need to go into game options and turn off DLSS & turn it back on for it to set itself up again)

Got it. Thanks.
